tests/index.test.js

```diff
@@ -0,0 +1,37 @@
+        { text: "שלום! תראה את האתר הזה: https://bit.ly/3xA 😃. זה פשוט מדהים! 🔥", expected: {encoding: 'UCS-2', numberOfSMS: 1, totalLength: 64, maxCharCount: 70}},
+        { text: "🔧✨ Tech at its finest! Δελτίο ενημέρωσης: https://bit.ly/3xA 🌟", expected: {encoding: 'UCS-2', numberOfSMS: 1, totalLength: 64, maxCharCount: 70}},
+        { text: "สวัสดีครับ! เยี่ยมชมเว็บไซต์นี้ได้ที่ https://bit.ly/3xA 🎉 สุดยอดเลย!", expected: {encoding: 'UCS-2', numberOfSMS: 1, totalLength: 70, maxCharCount: 70}},
+        { text: "Bonjour! Comment ça va? Ce site est incroyable: https://bit.ly/3xA 😁💯", expected: {encoding: 'UCS-2', numberOfSMS: 2, totalLength: 71, maxCharCount: 67}},
+        { text: "🎉🎂🥳🔥💥✨🙌💯", expected: {encoding: 'UCS-2', numberOfSMS: 1, totalLength: 15, maxCharCount: 70}},
+        { text: "Hello! How are you today? Just wanted to check in and say hi. Let's catch up soon. Take care!", expected: {encoding: 'GSM-7', numberOfSMS: 1, totalLength: 93, maxCharCount: 160}},
+        { text: "Hey! Don’t forget to bring the £5 you owe me. We’ll grab a coffee ☕ soon.", expected: {encoding: 'UCS-2', numberOfSMS: 2, totalLength: 73, maxCharCount: 67}},
+        { text: "Salut! Comment ça va? J’espère que tout va bien avec toi et ta famille. On se voit ce week-end?", expected: {encoding: 'UCS-2', numberOfSMS: 2, totalLength: 95, maxCharCount: 67}},
+        { text: "Hallo! Ich hoffe, es geht dir gut. Wir sollten uns bald wieder treffen und über alte Zeiten sprechen.", expected: {encoding: 'GSM-7', numberOfSMS: 1, totalLength: 101, maxCharCount: 160}},
+        { text: "¡Hola! ¿Cómo estás? Tengo muchas ganas de verte pronto. No olvides traer los documentos.", expected: {encoding: 'UCS-2', numberOfSMS: 2, totalLength: 88, maxCharCount: 67}},
+        { text: "The price is €100, and we need to deliver it by Monday at the latest. Don't forget!", expected: {encoding: 'GSM-7', numberOfSMS: 1, totalLength: 84, maxCharCount: 160}},
+        { text: "The total cost is €50. We'll need to confirm the payment by tomorrow morning.", expected: {encoding: 'GSM-7', numberOfSMS: 1, totalLength: 78, maxCharCount: 160}},
+        { text: "This message is designed to be exactly 160 characters long. No more, no less. Let's make sure every word counts in this message and that it stays within the limit.", expected: {encoding: 'GSM-7', numberOfSMS: 2, totalLength: 163, maxCharCount: 153}},
+        { text: "This is a long message that will exceed the 160 character limit for a single SMS message. When the length of this message exceeds the limit, it will be split into two or more parts.", expected: {encoding: 'GSM-7', numberOfSMS: 2, totalLength: 181, maxCharCount: 153}},
+        { text: "Hello, John! Please confirm the date & time for our meeting at 10:00 AM. Call me at 555-1234. Thanks!", expected: {encoding: 'GSM-7', numberOfSMS: 1, totalLength: 101, maxCharCount: 160}},
+    ];
```

Comment:
NIT: could you format this object for better readability?

Also I'm not seeing a test for the newline character (`\n`) could you add one pls?